src/__tests__/list-test.tsx

```diff
@@ -103,15 +103,16 @@ test('Row list with radio buttons', () => {
         <ThemeContextProvider theme={makeTheme()}>
             <RadioGroup name="radio-group">
                 <RowList>
-                    <Row title="Banana" radioValue="banana" />
-                    <Row title="Apple" radioValue="apple" />
+                    <Row title="Banana" subtitle="bananabanana" radioValue="banana" />
+                    <Row title="Apple" subtitle="appleapple" radioValue="apple" />
                 </RowList>
             </RadioGroup>
         </ThemeContextProvider>
     );
 
     const radioBanana = screen.getByRole('radio', {name: 'Banana'});
```

Comment:
This was working by coincidence because there was no subtitle and the whole text of the element was matching the selector